Dockerfile.x86_64

```diff
@@ -94,7 +94,7 @@ RUN curl -sSL https://www.openssl.org/source/openssl-$SSL_VER.tar.gz | tar xz &&
     cd openssl-$SSL_VER && \
     ./Configure no-zlib no-shared -fPIC --prefix=$PREFIX --openssldir=$PREFIX/ssl linux-x86_64 && \
     env C_INCLUDE_PATH=$PREFIX/include make depend 2> /dev/null && \
-    make -j$(nproc) && make install && \
+    make -j$(nproc) && make all install_sw && \
```

Comment:
Small optimization: `install_sw` skips doc/manpages generation and runs much faster (especially on the emulated ARM build).